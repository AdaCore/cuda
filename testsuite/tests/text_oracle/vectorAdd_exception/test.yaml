driver: text_oracle
expect_failure: True
