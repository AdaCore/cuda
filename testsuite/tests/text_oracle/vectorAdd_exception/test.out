[Vector addition of  4096 elements]
CUDA kernel launch with  16 blocks of  257  threads
Copy output data from the CUDA device to the host memory
kernel.adb:15: : block: [15,0,0], thread: [256,0,0] Assertion `` failed.
kernel.adb:15: : block: [15,0,0], thread: [241,0,0] Assertion `` failed.
kernel.adb:15: : block: [15,0,0], thread: [242,0,0] Assertion `` failed.
kernel.adb:15: : block: [15,0,0], thread: [243,0,0] Assertion `` failed.
kernel.adb:15: : block: [15,0,0], thread: [244,0,0] Assertion `` failed.
kernel.adb:15: : block: [15,0,0], thread: [245,0,0] Assertion `` failed.
kernel.adb:15: : block: [15,0,0], thread: [246,0,0] Assertion `` failed.
kernel.adb:15: : block: [15,0,0], thread: [247,0,0] Assertion `` failed.
kernel.adb:15: : block: [15,0,0], thread: [248,0,0] Assertion `` failed.
kernel.adb:15: : block: [15,0,0], thread: [249,0,0] Assertion `` failed.
kernel.adb:15: : block: [15,0,0], thread: [250,0,0] Assertion `` failed.
kernel.adb:15: : block: [15,0,0], thread: [251,0,0] Assertion `` failed.
kernel.adb:15: : block: [15,0,0], thread: [252,0,0] Assertion `` failed.
kernel.adb:15: : block: [15,0,0], thread: [253,0,0] Assertion `` failed.
kernel.adb:15: : block: [15,0,0], thread: [254,0,0] Assertion `` failed.
kernel.adb:15: : block: [15,0,0], thread: [255,0,0] Assertion `` failed.

raised CUDA.EXCEPTIONS.ERROR_CUDAERRORASSERT
>>>program returned status code 1
